docs/implement/authentication/redirect-guest-to-cas.md

```diff
@@ -1,7 +1,17 @@
 # Redirecting Guests to CAS for Sign-In
 
 An occasionally feature request is for uPortal to only support authenticated users,
-skipping a guest experience all together.
+skipping a guest experience all together. There are two solutions depending on which version of uPortal is in use.
```

Comment:
@bjagg -- I'm pretty sure the repo-housed documentation within the `docs/` directory is uP5-only.  The Confluence-based manuals are for previous versions.

Reply:
Then what do you think about removing the doc?

Reply:
Exactly -- I'd say just cover the uPortal 5 way as if it's the only way.